tests/e2e/main_test.go

```diff
@@ -66,6 +66,11 @@ func TestMain(m *testing.M) {
 		"",
 		"DCGM-exporter image tag to use")
 
+	flag.StringVar(&suiteCfg.arguments,
+		"arguments",
+		"",
+		"DCGM-exporter command line")
```

Comment:
Would be nice to given an example in the comments, how you expect a sample argument to look like esp if you have 2-3 arguments.